web/src/components/organisms/Workspace/index.tsx

```diff
@@ -0,0 +1,41 @@
+import ProjectCreationModal from "@reearth-cms/components/molecules/Common/ProjectCreationModal";
+import ProjectList from "@reearth-cms/components/molecules/ProjectList/ProjectList";
+import WorkspaceWrapper from "@reearth-cms/components/molecules/Workspace";
+import Greeting from "@reearth-cms/components/molecules/Workspace/Greeting";
+import WorkspaceHeader from "@reearth-cms/components/molecules/Workspace/WorkspaceHeader";
+
+import useHooks from "./hooks";
+
+export type Props = {
+  onWorkspaceModalOpen?: () => void;
+};
+
+const Workspace: React.FC<Props> = ({ onWorkspaceModalOpen }) => {
+  const {
+    handleProjectSearch,
+    handleProjectCreate,
+    handleProjectModalClose,
+    handleProjectModalOpen,
+    projectModalShown,
+    projects,
+  } = useHooks();
+
+  return (
+    <WorkspaceWrapper>
+      <Greeting />
+      <WorkspaceHeader
+        onProjectSearch={handleProjectSearch}
+        onProjectModalOpen={handleProjectModalOpen}
+        onWorkspaceModalOpen={onWorkspaceModalOpen}
+      />
+      <ProjectList projects={projects} handleProjectModalOpen={handleProjectModalOpen} />
+      <ProjectCreationModal
+        open={projectModalShown}
+        onClose={handleProjectModalClose}
+        onSubmit={handleProjectCreate}
+      />
```

Comment:
Can you import all this in to the molecule? That way the responsibilities are a little more spaced out (logic in organism, page make up/styles in the molecule).

Reply:
@KaWaite , this isn't a part of the Dashboard menu refactoring.
I prefer to keep this for another PR.